### PR TITLE
Added an explanation for using the double dash symbol

### PR DESCRIPTION
**Pull request by MoHD20**
I didn't get the idea of the double dash "--" at the first glance, So I though something like that might be added to the docs. I could add an additional example to pass the idea more instead of  just a simple note if that is preferable.

### CODE REVIEW

**Comment by MoHD20** [comment]
done signing a Contributor License Agreement (CLA).

File: content/en/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Get a shell to the running Container:
 ```shell
 kubectl exec -it shell-demo -- /bin/bash
 ```
+**Note:** The double dash symbol "--" is used to separate the arguments you want to pass to the command from the kubectl arguments.
```
Comment (lucperkins) [inline content/en/docs/tasks/debug-application-cluster/get-shell-running-container.md]: Could you add this as a note block instead? Like so:

```
{{< note >}}
The double dash symbol "--" is used to separate the arguments you want to pass to the command from the kubectl arguments.
{{< /note >}}
```

**Comment by steveperry-53** [comment]
/approve
@lucperkins Can you give this an `/lgtm` when you think it's ready.

**Comment by MoHD20** [comment]
Here is the thing, If you want to just run a program from the container you don't need the double dash symbol, It will just work without it. But in the other hand, If you tried to run something with an argument like 
`kubectl exec -it --namespace=tools some-pod -- bash -c "ls /home"`
You will need to add the double dash symbol for it to actually work. Hence "the command"

**Comment by steveperry-53** [comment]
@MoHD20 Sounds good, thanks.
As soon as @lucperkins gives this an LGTM, it will get merged.

**Pull request merged**